src/hearings/converters/number-of-attendances.converter.ts

```diff
@@ -0,0 +1,14 @@
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import * as fromHearingStore from '../store';
+import { AnswerConverter } from './answer.converter';
+
+export class NumberOfAttendancesConverter implements AnswerConverter {
```

Comment:
To keep consistent, could this number-of-attendances.converter.ts be renamed to  number-of-attendances.answer.converter.ts, and NumberOfAttendancesAnswerConverter, and the spec file as well?